client/src/main/java/io/split/client/SplitFactoryImpl.java

```diff
@@ -280,8 +291,22 @@ protected SplitFactoryImpl(String apiToken, SplitClientConfig config, CustomStor
 
         _evaluator = new EvaluatorImp(userCustomSplitAdapterConsumer, userCustomSegmentAdapterConsumer);
         _impressionsSender = PluggableImpressionSender.create(customStorageWrapper);
+
+        if (config.impressionsMode().equals(ImpressionsManager.Mode.NONE)){
+            int uniqueKeysRefreshRate = config.operationMode().equals(OperationMode.STANDALONE) ? config.uniqueKeysRefreshRateInMemory()
+                    : config.uniqueKeysRefreshRateRedis();
+            uniqueKeysTracker = new UniqueKeysTrackerImp(_telemetrySynchronizer, uniqueKeysRefreshRate, config.filterUniqueKeysRefreshRate());
+        }
```

Comment:
Maybe this could be extracted to a method? since it's being used in both constructors.
```suggestion
        uniqueKeysTracker = buildUniqueKeysTracker(_telemetrySynchronizer, config);
```

Reply:
I created a method for this. Thank you!